pkg/label/label.go

```diff
@@ -9,14 +9,26 @@ const (
 	// LegacyClusterID is an old style label for ClusterID
 	LegacyClusterID = "clusterID"
 
+	// LegacyClusterKey is an old style label to specify type of a secret that
+	// is used for guest cluster. This is replaced by RandomKey.
+	LegacyClusterKey = "clusterKey"
+
 	// LegacyComponent is an old style label to identify which component a
 	// specific CertConfig belongs to.
 	LegacyComponent = "clusterComponent"
 
+	// RandomKeyTypeEncryption is a type of randomkey secret used for guest
+	// cluster.
+	RandomKeyTypeEncryption = "encryption"
+
 	// ManagedBy label denotes which operator manages corresponding resource.
 	ManagedBy = "giantswarm.io/managed-by"
 
 	// Organization label denotes guest cluster's organization ID as displayed
 	// in the front-end.
 	Organization = "giantswarm.io/organization"
+
+	// RandomKey label specifies type of a secret that is used for guest
+	// cluster.
+	RandomKey = "giantswarm.io/randomkey"
```

Comment:
The label isn't used yet. Is it intentional?

Reply:
Oh shoot! No. Thanks for a good catch. I somehow got distracted for changing the old value key to suit for both old and new label key and then wondering if its location is correct in `label`. Will add that right away :flushed: